Devscord.DiscordFramework/Commons/Extensions/StringExtensions.cs

```diff
@@ -1,18 +1,12 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Text;
-
-namespace Devscord.DiscordFramework.Commons.Extensions
+﻿namespace Devscord.DiscordFramework.Commons.Extensions
 {
     public static class StringExtensions
     {
         public static string CutStart(this string value, string toCut)
         {
-            if(string.IsNullOrEmpty(toCut))
-            {
-                return value;
-            }
-            return value.Substring(toCut.Length, value.Length - toCut.Length);
+            return string.IsNullOrEmpty(toCut) 
```

Comment:
a co jeśli tekst nie zawiera tego tekstu albo jest on ale w środku a nie na początku?

lepiej zrobić to regexem 
```
$@"^{toCut}"
```

Reply:
```csharp
Regex.Replace(value, $@"^{toCut}", string.Empty, RegexOptions.Compiled)
```
jak dobrze pamiętam

Reply:
Przecież ja tu w logice nic nie zmieniłem. Jedynie uprościłem zapis. Mogę dać regexa